src/api/user/user.entity.ts

```diff
@@ -0,0 +1,13 @@
+import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn, UpdateDateColumn } from 'typeorm';
```

Comment:
Keep an eye on these warnings. good isntinct to include these columns it's generally a best practice to have timestamps for update, create, and delete.